website/content/en/docs/installation/install-operator-sdk.md

```diff
@@ -1,130 +1,106 @@
-
-[docker-tool]:https://docs.docker.com/install/
-[kubectl-tool]:https://kubernetes.io/docs/tasks/tools/install-kubectl/
-
 ## Install from Homebrew (macOS)
 
 If you are using [Homebrew][homebrew_tool], you can install the SDK CLI tool with the following command:
 
 ```sh
-$ brew install operator-sdk
+brew install operator-sdk
 ```
 
 ## Install from GitHub release
 
-### Download the release binaries
+#### Prerequisites
+
+- [curl](https://curl.haxx.se/)
+- [gpg](https://gnupg.org/) version 2.0+
+
+#### 1. Download the release binary
+
+Set platform information:
 
 ```sh
-# Set the release version variable
-$ RELEASE_VERSION=v1.1.0
-# Linux
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/ansible-operator-${RELEASE_VERSION}-x86_64-linux-gnu
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/helm-operator-${RELEASE_VERSION}-x86_64-linux-gnu
-# macOS
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/operator-sdk-${RELEASE_VERSION}-x86_64-apple-darwin
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/ansible-operator-${RELEASE_VERSION}-x86_64-apple-darwin
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/helm-operator-${RELEASE_VERSION}-x86_64-apple-darwin
+export ARCH=$(case $(arch) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(arch) ;; esac)
+export OS=$(uname | awk '{print tolower($0)}')
 ```
 
-#### Verify the downloaded release binaries
+Download the binary for your platform:
 
 ```sh
-# Linux
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/operator-sdk-${RELEASE_VERSION}-x86_64-linux-gnu.asc
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/ansible-operator-${RELEASE_VERSION}-x86_64-linux-gnu.asc
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/helm-operator-${RELEASE_VERSION}-x86_64-linux-gnu.asc
-# macOS
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/operator-sdk-${RELEASE_VERSION}-x86_64-apple-darwin.asc
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/ansible-operator-${RELEASE_VERSION}-x86_64-apple-darwin.asc
-$ curl -LO https://github.com/operator-framework/operator-sdk/releases/download/${RELEASE_VERSION}/helm-operator-${RELEASE_VERSION}-x86_64-apple-darwin.asc
+export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/latest/download
+curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH}
```

Comment:
Not working. properly for mac. 

export ARCH=$(case $(arch) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(arch) ;; esac)
export OS=$(uname | awk '{print tolower($0)}')
export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/latest/download
$ curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH}
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   166  100   166    0     0    260      0 --:--:-- --:--:-- --:--:--   261
100     9  100     9    0     0     10      0 --:--:-- --:--:-- --:--:--  9000

The file downloaded was `operator-sdk_darwin_i386 ` when should be `amd64`.

Also, after chmod +x I am facing  ./operator-sdk_darwin_i386 
./operator-sdk_darwin_i386: line 1: Not: command not found

Reply:
These binaries don't exist remotely yet, so this shouldn't work at all. On your machine what's the output of
```sh
arch
uname -a
```

Reply:
````
camilamacedo@Camilas-MacBook-Pro ~/go/src/sigs.k8s.io/kubebuilder/testdata/project-v2 (tets) $ arch
i386
camilamacedo@Camilas-MacBook-Pro ~/go/src/sigs.k8s.io/kubebuilder/testdata/project-v2 (tets) $ uname -a
Darwin Camilas-MacBook-Pro 18.7.0 Darwin Kernel Version 18.7.0: Mon Apr 27 20:09:39 PDT 2020; root:xnu-4903.278.35~1/RELEASE_X86_64 x86_64
````

It might need to be such as : 

OS=$(shell uname -s | tr '[:upper:]' '[:lower:]')
ARCH=$(shell uname -m | sed 's/x86_64/amd64/')

Reply:
>The file downloaded was operator-sdk_darwin_i386 when should be amd64.

>camilamacedo@Camilas-MacBook-Pro ~/go/src/sigs.k8s.io/kubebuilder/testdata/project-v2 (tets) $ arch
i386

The command is correct for your arch.